File: frontend/src/features/actividades/generar_recursos_pagos_cbr/models/generar_recursos_pagos_cbr.ts
export interface GenerarRecursosPagosCbr {
  id_generar_recursos_pagos_cbr: number;
  id_expediente: number;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/generar_recursos_pagos_cbr.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class generar_recursos_pagos_cbr : base_auditoria
    {
        public int id_generar_recursos_pagos_cbr { get; set; }
        public long id_expediente { get; set; }
        public string? observaciones { get; set; }
    }
}
